### PR TITLE
Enhance openstack init documentation

### PR DESCRIPTION
**Pull request by hoegaarden**
- Link bosh variables with
  - terraform outputs
  - Openstack RC files / variables
- Add other minor hints and tips

### CODE REVIEW

**Comment by hoegaarden** [comment]
CLA signed and sent via email.

**Comment by hoegaarden** [comment]
CLA has been accepted and filed, everything should now be set up to allow my contributions.
Let me know if there is anything else I need to do.

File: init-openstack.html.md.erb [hunk 1]
```diff
@@ -151,28 +151,36 @@ You must create and configure two Security Groups to restrict incoming network t
     $ git clone https://github.com/cloudfoundry/bosh-deployment
 
     # Fill below variables (replace example values) and deploy the Director
+    $ DIRECTOR_NAME='bosh-1'     # arbitrary name for the director
```
Comment (cppforlife) [inline init-openstack.html.md.erb]: how about putting comments inline? i think extra indirection thru vars is not really helpful.

Comment (hoegaarden) [inline init-openstack.html.md.erb]: I wanted to do that initially. However, when breaking long lines with `\` one cannot add comments afterwards. Also, I wanted to keep the code sample c&p-able.

One way I see is e.g.:
```
command \
    # a comment on arg \
    -some arg \
    # a comment on other_arg \
    -some other_arg
```

Just let me know what you'd prefer and I can adopt my change.

Comment (hoegaarden) [inline init-openstack.html.md.erb]: One thing I could do (which I personally do not like that much):
```
command \
    `# --first is very important` \
    --first arg \
    `# --second is super important too` \
    --second arg
```
Any better way to do inline comments properly?

Comment (cppforlife) [inline init-openstack.html.md.erb]: @hoegaarden how about this format: https://github.com/cloudfoundry/docs-bosh/blob/master/init-vsphere.html.md.erb#L43 ?

Comment (hoegaarden) [inline init-openstack.html.md.erb]: @cppforlife Sure, I'll rework the PR as soon as possible. Thanks for your input.

**Comment by hoegaarden** [comment]
@cppforlife does that look ok to you?

**Comment by cppforlife** [comment]
@hoegaarden just built https://github.com/cppforlife/js-annotate to make explanations a bit more easier to read. will try to use it for this page.

**Pull request closed without merging**